### ISSUE TITLE
able to download a website historically while only saving the 1st successful page?

### ISSUE THREAD

**Issue by devinschumacher** [jsvine/waybackpack#70]
any change to get a feature where we can download a site from a range of dates? for example 2015-Today to try and get every copy of a URL, but only save the most successful download?

the use case is im trying to get a website, but some pages are "blocked by cloudflare" on certain versions of archive.org

thanks!

**Comment by jsvine** [comment]
I don't think `waybackpack` currently supports this, but would be open to a PR that adds it. One tricky bit might be defining a criteria for "successful", particularly if the HTTP status code does not make it clear.

**Comment by devinschumacher** [comment]
> I don't think `waybackpack` currently supports this, but would be open to a PR that adds it. One tricky bit might be defining a criteria for "successful", particularly if the HTTP status code does not make it clear.

yeah i was thinking that. it would be a series of patterns prolly. what i normally see are things like `Cloudflare`, `Login`, `Too Many Requests` etc.
